internal/services/network/virtual_hub_resource_test.go

```diff
@@ -203,3 +221,19 @@ resource "azurerm_virtual_wan" "test" {
 }
 `, data.RandomInteger, data.Locations.Primary, data.RandomInteger)
 }
+
+func (r VirtualHubResource) hub_routing_preference(data acceptance.TestData) string {
```

Comment:
```suggestion
func (r VirtualHubResource) hubRoutingRreference(data acceptance.TestData) string {
```

Reply:
Hi @catriona-m , that's been fixed.

Thanks!